src/CashuWallet.ts

```diff
@@ -289,17 +320,21 @@ class CashuWallet {
 	/**
 	 * Request tokens from the mint
 	 * @param amount amount to request
-	 * @param hash hash to use to identify the request
+	 * @param hash hash to use to identify the request*
```

Comment:
Just a minor nit pick: Would it be possible to rename "hash" to "id" without breaking things since we're not using a hash of anything anymore? 

Afaik, in TS the name of the variable has no effect for the caller, right? I mean, implementors don't use `function(hash=<some_value>)` but only `function(<some_value>)`.

Reply:
makes sense, renamed to id